ammo-fe/public/index.html

```diff
@@ -22,17 +24,24 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
-    <!-- Roboto font for Material UI -->
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
-    <!-- Allow usage of Icon component of Material -->
-    <title>React App</title>
-</head>
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+        />
+        <!-- Roboto font for Material UI -->
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
+        />
+        <!-- Allow usage of Icon component of Material -->
+        <title>React App</title>
+        <script type="text/javascript" src="%PUBLIC_URL%/js/vh_fix.js"></script>
+    </head>
 
-<body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+    <body style="background-color: #250902">
```

Comment:
Because of complex CSS issues.. When on mobile you scroll up, the navigator bar disappears, slightly increasing the MainPannel size all of a sudden, leaving some blank space. With this workaround the white space is not visible during the resizing of the MainPannel